azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -126,7 +135,7 @@ def __init__(
                 "local": False,
                 "coupling_map": None,
                 "description": "IonQ QPU on Azure Quantum",
-                "basis_gates": ionq_basis_gates,
+                "basis_gates": GATESET_MAP[self.gateset()],
```

Comment:
```suggestion
                "basis_gates": self.gateset(),
```

Reply:
We discussed this offline, and we're keeping IonQ's convention.